cmd/disk-cache-utils.go

```diff
@@ -147,6 +147,10 @@ func readCacheFileStream(filePath string, offset, length int64) (io.ReadCloser,
 		return nil, err
 	}
 
+	if err = os.Chtimes(filePath, time.Now(), st.ModTime()); err != nil {
+		return nil, err
```

Comment:
Just a question/doubt - Are only PathErrors expected here.. just in case to know if we need to revert the time setting in case of error. Or `os.Chtimes` never updates the time incase of errors?

Reply:
only path errors are expected - [see](https://golang.org/src/os/file_posix.go?s=3996:4061#L135)